crates/ruma-client-api/src/error/kind_serde.rs

```diff
@@ -206,6 +216,9 @@ impl<'de> Visitor<'de> for ErrorKindVisitor {
             },
             ErrCode::CannotLeaveServerNoticeRoom => ErrorKind::CannotLeaveServerNoticeRoom,
             ErrCode::WeakPassword => ErrorKind::WeakPassword,
+            ErrCode::UnableToAuthorizeJoin => ErrorKind::UnableToAuthorizeJoin,
+            ErrCode::UnableToGrantJoin => ErrorKind::UnableToGrantJoin,
+            ErrCode::BadAlias => ErrorKind::BadAlias,
```

Comment:
Why were these added?

Reply:
Because I noticed they were missing in serde. That's why I put them in a separate commit.